src/Console/Command/ReleaseCommand.php

```diff
@@ -0,0 +1,318 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DevKit\Console\Command;
+
+use Github\Exception\ExceptionInterface;
+use Github\Exception\RuntimeException;
+use Packagist\Api\Result\Package;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+
+/**
+ * @author Jordi Sala <jordism91@gmail.com>
+ */
+final class ReleaseCommand extends AbstractCommand
+{
+    private static $labels = array(
+        'patch' => 'blue',
+        'bug' => 'red',
+        'docs' => 'yellow',
+        'minor' => 'green',
+        'pedantic' => 'cyan',
+    );
+
+    private static $stabilities = array(
+        'patch' => 'blue',
+        'minor' => 'green',
+        'pedantic' => 'yellow',
+    );
+
+    /**
+     * {@inheritdoc}
```

Comment:
Nice comment :trollface:

Reply:
Don't worry, I will die before this gets merged :trollface: